### ISSUE TITLE
Logged in user's view should have Drafts overview

### ISSUE THREAD

**Issue by igalic** [Plume-org/Plume#485]
When logged in, it would be good if my user's view `https://plu.me/me` should have a tab of my "Drafts" too.

**Comment by y6nH** [comment]
Agreed. I can never remember how to get to drafts, and just click around until I find them. The ability to quickly get back to what you were last working on is important.
